nservicebus/upgrades/azure-functions-service-bus-in-process-isolated-worker.md

```diff
@@ -146,3 +146,27 @@ class HttpSender
     }
 }
 ```
+
+## Ensuring consistency in the Isolated Worker model
+
+It's important to be able to maintain the consistency guarantees that were previously available in the in-process model when using `SendsAtomicWithReceive`.  To achieve this, there are a few options:
```

Comment:
Should we spell out that this is relevant IF `SendsAtomicWithReceive` is used in the in-process model? (Users have to opt-in to use it)